src/porepy/models/fluid_mass_balance.py

```diff
@@ -208,9 +217,15 @@ def fluid_flux(self, subdomains: list[pp.Grid]) -> pp.ad.Operator:
 
         """
         discr = self.mobility_discretization(subdomains)
-        mob_rho = self.fluid_density(subdomains) * self.mobility(subdomains)
-
-        bc_values = self.bc_values_mobrho(subdomains)
+        mob_rho = self.mobility_rho(subdomains)
+
+        boundary_projection = pp.ad.BoundaryProjection(self.mdg, subdomains=subdomains)
+        # NOTE Here (in the background), some Dirichlet-type data for p and T have
```

Comment:
This does not seem to me the right plase for this comment.

Reply:
Should I put it into a Note-sections in the docstring?

Reply:
Yes, I think that'd be better

Reply:
Alright.
Did the same for the enthalpy flux in `energy_balance.EnergyBalanceEquations`
Can you think of another another such case somewhere in pp?

Reply:
No, I don't think there are more advective fluxes at the moment.